querydsl-apt/src/main/java/com/querydsl/apt/AbstractQuerydslProcessor.java

```diff
@@ -610,6 +617,11 @@ private void serialize(Serializer serializer, Collection<EntityType> models) {
         }
     }
 
+    protected String getClassName(EntityType model) {
+        Type type = conf.getTypeMappings().getPathType(model, model, true);
+        String packageName = type.getPackageName();
+        return !packageName.isEmpty() ? (packageName + "." + type.getSimpleName()) : type.getSimpleName();
```

Comment:
Why negated? couldn't you just flip the branches?

Reply:
it's cut and paste from 570-572

Reply:
but sure, I'll flip them

Reply:
ahh, didn't catch that, but still it looks nicer if flipped

Reply:
fixed